tests/plots/test_force.py

```diff
@@ -2,9 +2,41 @@
 import pytest
 
 matplotlib.use('Agg')
-import shap  # pylint: disable=wrong-import-position
+import shap  # noqa: E402
 
 
+@pytest.mark.parametrize(
+    "cmap, emsg",
+    [
+        ("coolwarm", None),
+        (["#000000", "#ffffff"], None),
+        (777, "Plot color map must be string or list!"),
+        ([], "Color map must be at least two colors"),
+        (["#8834BB"], "Color map must be at least two colors"),
+        (["#883488", "#Gg8888"], r"Invalid color .+ found in cmap"),
+        (["#883488", "#1111119"], r"Invalid color .+ found in cmap"),
+    ],
+    ids=[
```

Comment:
A minor suggestion, for readability you can provide the IDs in-line with the test case like so:

```python
from pytest import param

@pytest.mark.parametrize(
    "cmap, emsg",
    [
        param("coolwarm", None, id="valid-str"),
        param(["#000000", "#ffffff"],

Reply:
oh nice!, didn't know this option existed